src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSService.java

```diff
@@ -510,4 +508,49 @@ RunTaskResult runEcsTask(final ECSSlave agent, final ECSTaskTemplate template, S
         return client.runTask(req);
     }
 
+    private static ContainerOverride getContainerOverride(Collection<String> command, String agentContainerName, KeyValuePair envNodeName,
+            KeyValuePair envNodeSecret, ECSTaskTemplate template, String jobName) {
+        var overrides = new ContainerOverride()
+                .withName(agentContainerName)
+                .withCommand(command)
+                .withEnvironment(envNodeName)
+                .withEnvironment(envNodeSecret);
+
+        if(template.getUseSessionTags()) {
```

Comment:
If we're using session tags, then assume the role we wanna add session tags to, and add them.
In this case, just the job name